src/main/java/frc/robot/GameRobotContainer.java

```diff
@@ -73,6 +76,12 @@ public GameRobotContainer() {
                                                         ClimbConstants.MOTOR_LEFT_INVERTED, 
```

Comment:
I think you wanted  `driverController.leftBumper().whileTrue(new DecreaseSpeed(closedFieldRel));` somewhere also